MIGRATIONS.unreleased.md

```diff
@@ -9,4 +9,7 @@ User-facing changes are documented in the [changelog](CHANGELOG.released.md).
 [Commits](https://github.com/scalableminds/webknossos/compare/25.05.0...HEAD)
 
 ### Postgres Evolutions:
+<<<<<<< HEAD
+
```

Comment:
:grimacing:

Reply:
Oops. I’ll repair it in https://github.com/scalableminds/webknossos/pull/8559